js/options.js

```diff
@@ -45,10 +45,14 @@ function restoreOptions() {
       document.optform.skritterTLD[0].selected = true;
     }
 
-    if (options.zhuyin == 'yes') {
-      document.optform.zhuyin[1].selected = true;
+    if (options.translit == 'zhuyin') {
+      document.optform.translit[1].selected = true;
+    } else if (options.translit == 'pinyin') {
+      document.optform.translit[2].selected = true;
+    } else if (options.translit == 'zhuyin_pinyin') {
+      document.optform.translit[3].selected = true;
     } else {
-      document.optform.zhuyin[0].selected = true;
+      document.optform.translit[0].selected = true;

```

Comment:
This effectively makes "none" the default if options.translit is not set. The default should be pinyin.